src/common/util/game_settings.hpp

```diff
@@ -25,6 +25,8 @@ const sf::Vector2f WINDOW_RESOLUTION(2688, 1512);
 const float GAME_SCALE = 1.f;
 const sf::Color BACKGROUND_COLOR(sf::Color::Black);
 
+const bool IS_BOT = false;
```

Comment:
I don't think this should be defined here.
This is a constants file that's shared between the client and the server

Maybe we can have a class that sits between `InputController` and `ClientGameController` that's responsible for determining input and that class can be initialized as a bot or a human. 🤷‍♂

Reply:
Yep, my plan here is to actually take the "mode" from the command line or a config file. so you would run `ug-client --mode=bot --strategy=greedy` something like that